learners/setup.md

```diff
@@ -69,6 +69,12 @@ After the installation is complete, activate the environment:
 ```bash
 mamba activate medical_image_proc
 ```
+Note: In the case that this setup does not work due to conflicting previous conda or mamba configurations on your machine, you can try cloning the repository, navigating into it, and then building the environment there.
+
+The cloning step is as follows: 
+```bash
+git clone https://github.com/esciencecenter-digital-skills/medical-image-processing.git
```

Comment:
What was the issue you encountered specifically? At which point did the setup break for you?
If it's here: `mamba env create -f https://github.com/esciencecenter-digital-skills/medical-image-processing/blob/main/learners/environment.yml` and it is due to mamba installation you can try to change `mamba` to `conda` (this is also indicated in the instructions, line 47). 

I don't understand how cloning the repository can solve any possible conda/mamba conflicts (after you clone it, you need to use conda or mamba conda anyway to create the env, and if the system has conflicts they will still be there).

Reply:
I tried changing the mamba for conda, and it still did not work. Something in my setup just does not work when I pull things this way, although I have no idea why. What did work was cloning the repo and going inside of it. Maybe the issue is not mamba/conda related, but I worked on this for a while, and it was a reproducible problem on my machine, at least the day I worked on it.